web/src/components/AddQuestion.jsx

```diff
@@ -184,13 +175,13 @@ function AddQuestion({ loading, gameIndex, game, saveGame, deleteQuestion, selec
       })}
       <Grid container className={classes.parent}>
         <Grid item xs={2}>
-          <Button className={classes.greenButton} color="primary" type="button" variant="contained" onClick={() => addQuestion(questions[index])}>Add to Game</Button>
+          <button className={classes.greenButton} color="primary" type="button" variant="contained" onClick={() => handleColoneQuestion(questions[index-1])}>Add to Game</button>
```

Comment:
nitpick: misspelling (only change if the misspelling of the actual function above on line 119 because those names need to match)